app/scripts/metamask-controller.js

```diff
@@ -76,6 +76,8 @@ import MetaMetricsController from './controllers/metametrics';
 import { segment } from './lib/segment';
 import createMetaRPCHandler from './lib/createMetaRPCHandler';
 
+const EXTENSION_CLIENT_ID = 'extension'
```

Comment:
I see also use this constant here: https://github.com/MetaMask/metamask-extension/blob/develop/ui/pages/swaps/swaps.util.js#L56

Could we make `EXTENSION_CLIENT_ID` a shared constant? Perhaps place it in `shared/constants/network.js`?

Reply:
Also, this line needs a `;` at the end

Reply:
I'm not sure if `shared/constants/network.js` makes sense because it's not network-specific. How about `shared/constants/app.js`? Or if that is too general, how about `shared/constants/swaps.js`?

Reply:
I think you're right that `shared/constants/swaps.js` makes the most sense!

Reply:
Awesome, pushed those changes. Let me know what you think.